app/common/containers/pages/MedicationDispensesListPage/index.js

```diff
@@ -67,6 +67,7 @@ const MedicationDispensesListPage = ({
 
     <div>
       <DateFilterForm
+        items={FILTER_DATE[0].names}
```

Comment:
Maybe use `activeDateFilter` here?

Reply:
@kiurchv  In case of empty query search we will have Fields without names

Reply:
In that case, we should have default filter as activeDateFilter. This behavior is implemented in getFilter function